packages/create-graz-app/package.json

```diff
@@ -29,9 +29,9 @@
     "@types/validate-npm-package-name": "^4.0.0",
     "async-retry": "1.3.3",
     "chalk": "^5.2.0",
-    "commander": "^11.0.0",
+    "commander": "^10.0.1",
     "cross-spawn": "^7.0.3",
-    "got": "^13.0.0",
+    "got": "^12.6.0",
```

Comment:
My bad, keep including the deps update on this. Did something break if using latest versions, @codingki?

Reply:
the build failed